src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceStringToArray = sourceString
```

Comment:
There is a syntax error here. The division operator `/` is incorrectly used instead of the assignment operator `=`. It should be `const sourceStringToArray = sourceString`.